x-pack/rest-resources-zip/build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
   freeTests project(path: ':rest-api-spec', configuration: 'restTests')
   freeTests project(path: ':modules:aggregations', configuration: 'restTests')
   freeTests project(path: ':modules:analysis-common', configuration: 'restTests')
+  freeTests project(path: ':modules:data-streams', configuration: 'basicRestSpecs')
```

Comment:
I only exposed these YAML tests in [1fd8a8b](https://github.com/elastic/elasticsearch/commit/1fd8a8ba34cec876d6f98c84a8fe25b03bd6e1d7]) for the multi-project work, and these changes are temporary - in the future we'll run these tests in MP mode directly, instead of including them in a dedicated MP suite. Therefore, I'm inclined to say we shouldn't have this line here. However, that does beg the question of whether these tests are currently included or not. Do you know if there's a way we can determine whether that's the case? TBH, I'm not really sure what this `rest-resources-zip` is used for exactly. A lot of modules seem to be missing here.

Reply:
Thanks, done!